### DISCUSSION TITLE
Is there anyway to change nation mapcolor in 1.7.10

### DISCUSSION THREAD

**Discussion by Shaw9682** [TownyAdvanced/Towny#7494] [Q&A] [answered]
Hello!
I'm a admin of a towny server, our admin groups and server owner are confused on how to change nation mapcolor for like years, because we tried many way to change our towny mapcolor(We've seen a topic about how to change the mapcolor before, the /n set mapcolor of course.But it seems doesn't work in that version or what else), but it all failed and ended up still using the original red color. Also we've seen another 1.7 towny server using nations with different mapcolors.
Just asking is there anyway to change mapcolor in 0.92 version for 1.7.10

I uploaded a picture of map of the mentioned server, which explained they've colored countries' mapcolor.
![IMG_20240706_082121](https://github.com/TownyAdvanced/Towny/assets/174593752/69680818-8ea8-4c52-b826-4d8dc90bc0f8)

**Comment by LlmDl** [comment, marked as answer]
If the command doesn't work, no.

**Reply by Shaw9682** [reply]
Then is there a way to set town's default colors?

**Reply by LlmDl** [reply]
Not in old versions like that.
